pkg/manifest/manifest.go

```diff
@@ -3,9 +3,11 @@ package manifest
 import (
 	"bytes"
 	"fmt"
+	goyaml "gopkg.in/yaml.v2"
```

Comment:
nit: non-stdlib package, so ideally import goes down in the non-stdlib block below.